src/ResourceManager/Automation/ChangeLog.md

```diff
@@ -18,13 +18,14 @@
         - Additional information about change #1
 -->
 ## Current Release
-
+* Made changes to AutomationDSC* cmdlets to pull more than 100 records
+* Resolved the issue where the Verbose streams stop working after calling some Automation cmdlets (for example Get-AzureRmAutomationVariable, Get-AzureRmAutomationJob).
 ## Version 3.2.1
 
+>>>>>>> 299c84a7882ca09e064fc82d51ff75bbfffbd275
```

Comment:
Yes, if the bug is that you are not returnign all pages, then you need a test that actually presents multiple pages.  The tests you have included essentially only verify that the correct method is called,  You need to ensure that the correct method for listing the next page of results is called when there is a next page